src/views/ManageApartments.vue

```diff
@@ -202,6 +219,14 @@ export default defineComponent({
       ];
     };
 
+    const displayApartmentModal = computed(() => {
+      return store.state.cooperationStore.displayModal;
```

Comment:
Do you use this store property in any other component?

Reply:
fixed